app/templates/containerdetails.html

```diff
@@ -361,7 +361,7 @@ <h3 class="pull-right">
             <strong uib-tooltip="Date and time of the last update to the Tool entry">Last Update</strong>:
             {{ containerObj.lastUpdated ? getDateTimeString(containerObj.lastUpdated) : 'n/a' }}
           </li>
-          <li>
+          <li ng-show="containerObj.description">
```

Comment:
One addition to this. 
Can we add a warning in when looking at a tool in "My Tools" to add a description if one is missing? 
Thanks